src/generate.ts

```diff
@@ -214,7 +214,11 @@ function extractFromZodSchema<ZSchema extends ZodTypeAny>(
 		ZodAny: () => request('any'),
 		ZodUnknown: () => request('unknown'),
 		ZodNever: () => request('never'),
-		ZodEffects: () => request(extractFromZodSchema(schema._def.schema).type),
+		ZodEffects: () => {
```

Comment:
Hi, thanks for raising this PR.
This reminds me that an effect is not invoked.
What do you think about changing this PR so that does happen?
If you want

Reply:
Done ✨ 

Now ZodEffects with `type: 'transform'` are invoked. However, I'm not very sure about these lines: https://github.com/timdeschryver/zod-fixture/blob/7a9b801f68c6da56fb85b931fbc13b59119bf025/src/customizations/effect-customization.ts#L25-L27

This means all issues that were added in the transformation function will be ignored. Don't know if we should handle them somehow?

Reply:
Nice 🕺
I think this is fine for now, feel free to add issues in a separate PR if you need that feature.
Thanks for your contribution! 🏆